### DISCUSSION TITLE
Annual vs 2.0 Changes Due Date

### DISCUSSION THREAD

**Discussion by madhhc** [CMSgov/hospital-price-transparency#30] [Q&A] [answered]
Hospitals are required to post annual updated files to their website(s).  CMS requires version 2.0 changes to be implemented by July 1, 2024.  Does this mean that **_ALL_** hospitals must repost their files on July 1, 2024, or only on their annual due date?

**Comment by RidenourK** [comment, marked as answer]
@madhhc. Thank you for your question. 

As finalized in the CY2024 OPPS/ASC final rule, beginning July 1, 2024, all hospitals must adopt a CMS template and encode its standard charge information in it in the form and manner finalized in the [Hospital Price Transparency – Data Dictionary GitHub repository](https://github.com/CMSgov/hospital-price-transparency). CMS intends to begin enforcing these requirements on July 1, 2024.
